### ISSUE TITLE
Replace file() with open() in twisted/python

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8523]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)| @rodrigc reported|
|-|-|
|Trac ID|trac#8523|
|Type|enhancement|
|Created|2016-06-28 06:17:57Z|
|Branch|https://github.com/twisted/twisted/tree/8523-rodrigc-python|

In Python 3, file() is gone (see https://docs.python.org/3/whatsnew/3.0.html#builtins ), and needs to be replaced with open()
which works on Python 2 and 3

<details><summary>Searchable metadata</summary>

```
trac-id__8523 8523
type__enhancement enhancement
reporter__rodrigc rodrigc
priority__normal normal
milestone__Python_3_x Python-3.x
branch__8523_rodrigc_python 8523-rodrigc-python
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1467094677565413 1467094677565413
changetime__1467114046486099 1467114046486099
version__None None
owner__rodrigc rodrigc

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_3"></a>|@rodrigc commented|
|-|-|

See:
https://github.com/twisted/twisted/pull/247

**Comment by twisted-trac** [comment]
|[<img alt="Lukasa's avatar" src="https://avatars.githubusercontent.com/u/1382556?s=50" width="50" height="50">](https://github.com/Lukasa)<a name="note_4"></a>|@Lukasa set owner to @rodrigc|
|-|-|

All green, patch looks good. Merge when ready.

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_5"></a>|GitHub set status to `closed`|
|-|-|

In changeset edac09817eb39ba4d625472cd9d6b3991ee3e35b
```
#!CommitTicketReference repository="" revision="edac09817eb39ba4d625472cd9d6b3991ee3e35b"
Merge 8523-rodrigc-python: Replace file() with open() in twisted/python

Author: rodrigc
Reviewer: Lukasa
Fixes: #8523
```
